src/SendGrid/Properties/AssemblyInfo.cs

```diff
@@ -1,14 +1,16 @@
-﻿// <copyright file="AssemblyInfo.cs" company="Twilio SendGrid">
-// Copyright (c) Twilio SendGrid. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-// </copyright>
-
-using System.Runtime.InteropServices;
-
-// Setting ComVisible to false makes the types in this assembly not visible
-// to COM components.  If you need to access a type in this assembly from
-// COM, set the ComVisible attribute to true on that type.
-[assembly: ComVisible(false)]
-
-// The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: Guid("377c20e4-2297-488f-933b-fb635c56d8fc")]
+﻿// <copyright file="AssemblyInfo.cs" company="Twilio SendGrid">
+// Copyright (c) Twilio SendGrid. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("377c20e4-2297-488f-933b-fb635c56d8fc")]
+[assembly: InternalsVisibleTo("SendGrid.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100812ec26a66c8e08c790704ac4b46bcc9da9f4bca4da0ec7c06ce6dcd73baeb2c5525f36a237b253e80e16febb4c052f50734d5e1cf3bf478d9c88f0f69df53b47306419182983bc35c33c3bafb5e90b9bd7aa7b9a9da09abe3667d50db891012e077e4b9aefe9799a58222fa67127c230219755d7670073c7463d90cf9e79dba")]
```

Comment:
How was this public key generated?

Reply:
The SendGrid assembly is signed using a strongly named key, it's the `tools/sendgrid-csharp.snk` key already in the repository. I just grabbed the public key token for the SNK.